### PR TITLE
Remove deprecated 'fixed' containers

### PR DESCRIPTION
**Pull request by domlander**
Corresponding [frontend PR](https://github.com/guardian/frontend/pull/28660)

## What does this change?

Removes the deprecated "fixed" container types:
- fixed/large/slow-XIV
- fixed/medium/fast-XI
- fixed/medium/fast-XII
- fixed/medium/slow-VI
- fixed/medium/slow-VII
- fixed/medium/slow-XII-mpu
- fixed/small/fast-VIII
- fixed/small/slow-I
- fixed/small/slow-III
- fixed/small/slow-IV
- fixed/small/slow-V-half
- fixed/small/slow-V-mpu
- fixed/small/slow-V-third

## Why?

These collections are no longer used anywhere following a migration ran by @abeddow91.

### CODE REVIEW

File: dotcom-rendering/fixtures/manual/frontCollections.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ const defaultGrouped = {
 
 const defaultValues = {
 	backfill: [],
-	collectionType: 'fixed/small/slow-IV',
+	collectionType: 'static/medium/4',
```
Comment (domlander) [inline dotcom-rendering/fixtures/manual/frontCollections.ts:12]: `fixed/small/slow-IV` & `fixed/small/slow-V-mpu` most closely resemble `static/medium/4` in the new world. All the others are most similar to the flexible `flexible/general` container.

See: [Chromatic](https://www.chromatic.com/library?appId=63e251470cfbe61776b0ef19&search=&groupPrefix=Front+Containers%2FDeprecated+Containers) for visuals

File: dotcom-rendering/src/lib/getFrontsAdPositions.test.ts [hunk 2]
```diff
@@ -350,13 +350,13 @@ describe('Desktop Ads', () => {
...
 
-		expect(adPositions).toEqual([3, 6, 8, 11, 14, 17, 20]);
+		expect(adPositions).toEqual([3, 6, 8, 14, 17]);
 	});
 
 	it('calculates ad positions correctly for an example of the US network front', () => {
 		const adPositions = getDesktopAdPositions(testCollectionsUs, 'us');
 
-		expect(adPositions).toEqual([3, 6, 8, 11, 14, 18, 20, 22]);
+		expect(adPositions).toEqual([3, 6, 10, 12, 19]);
```
Comment (arelra) [inline dotcom-rendering/src/lib/getFrontsAdPositions.test.ts:359]: Adding @guardian/commercial-dev to verify

Comment (domlander) [reply dotcom-rendering/src/lib/getFrontsAdPositions.test.ts:359]: The reason ads are now more sparse in this test is because the flexible general containers that have replaced deprecated containers [are empty](https://github.com/guardian/dotcom-rendering/blob/4f379d70b13832b250fd76dc9d7c5cc52100b883/dotcom-rendering/fixtures/manual/frontCollections.ts#L4) and I think therefore it will have zero height predicted. To make for a more realistic test, populate these groups with test cards.
